interface/billing/sl_eob_search.php

```diff
@@ -675,43 +676,46 @@ function toEncSummary(pid) {
-    } elseif ($GLOBALS['enable_help'] == 0) {
-        $help_icon = '';
-    }
-    ?>
-    <?php
-    //to determine and set the form to open in the desired state - expanded or centered, any selection the user makes will
-    //become the user-specific default for that page. collectAndOrganizeExpandSetting() contains a single array as an
-    //argument, containing one or more elements, the name of the current file is the first element, if there are linked
-    // files they should be listed thereafter, please add _xpd suffix to the file name
+    //BEGIN - edit as needed - variables needed to construct the array $arrHeading - needed to output the Heading text with icons and Help modal code
+    $name = " - " . getPatientNameFirstLast($pid); //un-comment to include fname lname, use ONLY on relevant pages :))
+    $heading_title = xlt('EOB Posting - Search') . $name; // Minimum needed is the heading text
+    //3 optional icons - for ease of use and troubleshooting first create the variables and then use them to populate the arrays:)
     $arr_files_php = array("sl_eob_search_xpd");
-    $current_state = collectAndOrganizeExpandSetting($arr_files_php);
-    require_once("$srcdir/expand_contract_inc.php");
+    $current_state= collectAndOrganizeExpandSetting($arr_files_php);
+    $expandable = 1;
+    $arrExpandable = array($current_state, $expandable);//2 elements - int|bool $current_state, int|bool $expandable . $current_state = collectAndOrganizeExpandSetting($arr_files_php).
+                            //$arr_files_php is also an indexed array, current file name first, linked file names thereafter, all need _xpd suffix, names to be unique
+    $action = 'reset';
+    $action_title = '';
+    $action_href = 'sl_eob_search.php';
+    $arrAction = array($action, $action_title, $action_href);//3 elements - string $action (conceal, reveal, search, reset, link and back), string $action_title - leave blank for actions
+                        // (conceal, reveal and search), string $action_href - needed for actions (reset, link and back)
+    $show_help_icon = 1;
+    $help_file_name = 'sl_eob_help.php';
+    $arrHelp = array($show_help_icon, $help_file_name );// 2 elements - int|bool $show_help_icon, string $help_file_name - file needs to exist in Documentation/help_files directory
+    //END - edit as needed
+    //DO NOT EDIT BELOW
+    $arrHeading = array($heading_title, $arrExpandable, $arrAction, $arrHelp); // minimum $heading_title, others can be an empty arrays - displays only heading
+    $oemr_ui = new OemrUI($arrHeading);
+    $arr_display_heading = $oemr_ui->pageHeading(); // returns an indexed array containing heading string with selected icons and container string value
+    $heading = $arr_display_heading[0];
+    $container = $arr_display_heading[1];// if you want page to always open as full-width override the default returned value with $container = 'container-fluid'
```

Comment:
for the above 3 lines better to have a separate function for each call and to just call it where it is needed in the script
(I will show below)